cocos/core/algorithm/easing.ts

```diff
@@ -197,7 +197,7 @@ export function sineInOut (k: number): number {
  * @zh 启动慢，加速快。具体效果可以参考[该文档](https://docs.cocos.com/creator/manual/zh/tween/tween-function.html)。
  */
 export function expoIn (k: number): number {
-    return k === 0 ? 0 : Math.pow(1024, k - 1);
+    return k === 0 ? 0 : 1024 ** (k - 1);
```

Comment:
Fix the error: 

'Math.pow' is restricted from being used. Use the exponentiation operator (**) instead.eslint[no-restricted-properties](https://eslint.org/docs/latest/rules/no-restricted-properties)
Use the '**' operator instead of 'Math.pow'.eslint[prefer-exponentiation-operator](https://eslint.org/docs/latest/rules/prefer-exponentiation-operator)